src/main/kotlin/com/mutualmobile/praxisspringboot/entities/projects/DBUserWork.kt

```diff
@@ -0,0 +1,19 @@
+package com.mutualmobile.praxisspringboot.entities.projects
+
+import com.mutualmobile.praxisspringboot.entities.BaseEntity
+import java.util.Date
+import javax.persistence.Entity
+import javax.persistence.Table
+
+@Entity
+@Table(name = "user_work")
+class DBUserWork(
+    // Used to connect this table to the user_project table
+    val userProjectId: String,
```

Comment:
@shubhamsinghmutualmobile we need the user id of the user along with the project id. to map in the table.

Reply:
We have a separate table for that (user_project). We're doing this so that we don't have multiple entries in the table with the same userId & projectId.